### PR TITLE
Pac death

### PR DESCRIPTION
**Pull request by rcbrowder**
1. Functions added
    * `deathReset()`
    * `resetBit()`
2. Functions updated
    * `animate()` - Parts taken out of animate and put into separate function `resetBit()`

### CODE REVIEW

File: app/main.js [hunk 1]
```diff
@@ -260,7 +271,7 @@ function checkForWin() {
 function checkForLoss() {
     if (lives == 0) {
         resetGame();
-        drawMessage(youWin);
+        drawMessage(gameOver);
```
Comment (emberborja) [inline app/main.js:274]: I was confused by the lives == 0, youWin message before. Why was it that way before and now it is gameOver?

File: app/main.js [hunk 2]
```diff
@@ -692,6 +696,17 @@ function animate() {
     }
 }
 
+function resetBit() {
```
Comment (emberborja) [inline app/main.js:699]: why is it "resetBit" ?

File: app/main.js (hunk 1 above)
Comment (rcbrowder) [reply app/main.js:274]: It was just a mistake. The `gameOver` variable should be in there like it is now.

**Review by KarKunze — APPROVED** [review]

File: app/main.js (hunk 2 above)
Comment (rcbrowder) [reply app/main.js:699]: The tiles that make up the maze walls come from a "bit map". The initial render of the walls is `drawBit()` and this function resets the walls around pac-man when he dies. I guess a better name would be `resetTile()` or `resetWall()`. I'll open an issue to remind us to go back and change function and variable names to be more accurate.

**Review by emberborja — APPROVED** [review]

**Pull request merged**